Cargo.toml

```diff
@@ -16,6 +16,7 @@ exclude = [
         "examples",
 ]
 resolver = "2"
+cargo-features = ["rename-dependency"]
```

Comment:
I don't think this is needed because we already use `package = ` in other places.